.github/workflows/flake_vendorhash.yaml

```diff
@@ -57,5 +57,8 @@ jobs:
           branch: nix/flake
           delete-branch: true
           sign-commits: true
+          labels: |
+            kind/skip-release-notes
+            chore
```

Comment:
which one will then be taken?

Reply:
both but since skip is mentioned it will be skipped completely